### PR TITLE
Add check in GlobalProperties for showing twitter in bookmark buttons.

### PR DESCRIPTION
**Pull request by Tamba32**
Connected with cbioportal/cbioportal-frontend PR: https://github.com/cBioPortal/cbioportal-frontend#727

### CODE REVIEW

File: portal/src/main/webapp/WEB-INF/jsp/global/frontend_config.jsp [hunk 1]
```diff
@@ -13,6 +13,7 @@ window.legacySupportFrontendConfig = {
     genomeNexusApiUrl : '<%=GlobalProperties.getGenomeNexusApiUrl()%>',
     showCivic : <%=GlobalProperties.showCivic()%>,
     showHotspot : <%=GlobalProperties.showHotspot()%>,
+    showTwitter: '<%=GlobalProperties.showTwitterBookmark()%>',
```
Comment (inodb) [inline portal/src/main/webapp/WEB-INF/jsp/global/frontend_config.jsp]: @Tamba32 @alisman remove quotes there to pass as boolean instead of string

Comment (Tamba32) [reply portal/src/main/webapp/WEB-INF/jsp/global/frontend_config.jsp]: @inodb @alisman thanks!

File: core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java [hunk 2]
```diff
@@ -140,6 +140,8 @@
     public static final String ONCOKB_PUBLIC_API_URL = "oncokb.public_api.url";
     public static final String SHOW_ONCOKB = "show.oncokb";
 
+    public static final String SHOW_TWITTER_BOOKMARK = "bookmark.twitterLink.enabled";
```
Comment (inodb) [inline core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java]: @Tamba32 Can you either
-  add the `@Value{$bookmark.twitter_link.enabled}` decorator? That way we can change the value on runtime with `-d bookmark.twitter_link.enabled=true` or `-d bookmark.twitter_link.enabled=false`. -- Or -
- another way might be to use  bookmark.twitter_link.enabled directly in the JSP w/o using GlobalProperties class (@ersinciftci might know).

Also for consistency you prolly don't want to use camelCase on the properties, they all use underscores and dots.

**Comment by jjgao** [comment]
Should this go to master branch?

**Pull request closed without merging**